src/common/components/traffic-lights/traffic-light.css

```diff
@@ -55,7 +55,7 @@ $signal-colour-default: $light-grey;
 }
 
 .box:last-child .signal::after {
-  display:none;
+  background: #FFF; /* cover any line overflowing from previous signals */
```

Comment:
Depends on the fact that we have white background, but at least it covers connector overflowing on smaller widths.